operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
 * Community and doc review process
   * Exact split with Justine to be determined
 
-Note: Few areas not listed above, such as the [User's Guide](https://docs.mattermost.com/guides/user.html), will remain a collaborative effort between Product Managers and Technical Writers. Please assume @jason.blais as the directly responsible individual.
+Note: Some areas not listed above, such as the [User's Guide](https://docs.mattermost.com/guides/user.html), will remain a collaborative effort between Product Managers and Technical Writers. Please assume @jason.blais as the directly responsible individual.
 
 ## Processes
 
 The processes around documentation, such as SLAs for issues and requesting documentation assistance are detailed below.
 
-### **Contributing to Documentation**
+### Contributing to Documentation
 
 The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
 
-Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/) - it applies to various types of contributions including documentation.
+Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
-### DocUp/Documentation Issue Process
+### Documentation MVP
+
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide. 
```

Comment:
`As such, we don't expect every contribution to adhere to our style guide. ` --> should we reword it differently to emphasize that you or someone else will update the contribution to adhere to our style guide. I don't want to set the wrong expectations about lowering our standards

Reply:
Yep, that's not properly conveyed at all.

How about:
"As such, we don't expect every contribution to adhere to our style guide when first submitted. During the review process the editorial team also provide feedback on style elements to bring the submission in line with the Documentation Style [Guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide)."